### PR TITLE
chore: reference colors for metadata icons

### PR DESCRIPTION
**Pull request by ChenCodes**
Changes in this PR:
- use constants from variables.js instead of hardcoding colors in MetadataActiveBadge / MetadataDefaultBadge

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ChenCodes has signed the CLA. Thanks for the pull request!

File: src/icons/badges/MetadataActiveBadge.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const MetadataActiveBadge = ({ className = '', height = 16, title, width = 16 }:
             <circle cx="8" cy="8" fill="none" r="7.5" stroke="#4D91E2" />
```
Comment (priyajeet) [inline src/icons/badges/MetadataActiveBadge.js]: #4D91E2 too

File: src/icons/badges/MetadataDefaultBadge.js [hunk 2]
```diff
@@ -19,10 +20,10 @@ const MetadataDefaultBadge = ({ className = '', height = 16, title, width = 16 }
         width={width}
     >
         <g fill="none" fillRule="evenodd">
-            <circle cx="8" cy="8" fill="none" r="7.5" stroke="#767676" />
+            <circle cx="8" cy="8" fill="none" r="7.5" stroke={bdlNeutral02} />
             <path
                 d="M5.126 10.545a.57.57 0 0 1-.693.437.603.603 0 0 1-.416-.728l1.143-4.8c.122-.513.772-.62 1.039-.172L8 8.308l1.801-3.026c.267-.449.917-.34 1.039.173l1.143 4.8a.603.603 0 0 1-.416.727.57.57 0 0 1-.693-.437l-.817-3.43-1.572 2.643a.557.557 0 0 1-.97 0L5.943 7.116l-.817 3.43z"
-                fill="#767676"
+                fill={bdlNeutral02}
```
Comment (priyajeet) [inline src/icons/badges/MetadataDefaultBadge.js]: Do not use neutrals, use bdlGray

File: src/icons/badges/MetadataActiveBadge.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ const MetadataActiveBadge = ({ className = '', height = 16, title, width = 16 }:
...
             <path
                 d="M5.126 10.545a.57.57 0 0 1-.693.437.603.603 0 0 1-.416-.728l1.143-4.8c.122-.513.772-.62 1.039-.172L8 8.308l1.801-3.026c.267-.449.917-.34 1.039.173l1.143 4.8a.603.603 0 0 1-.416.727.57.57 0 0 1-.693-.437l-.817-3.43-1.572 2.643a.557.557 0 0 1-.97 0L5.943 7.116l-.817 3.43z"
-                fill="#0061D5"
+                fill={boxBlue}
```
Comment (priyajeet) [inline src/icons/badges/MetadataActiveBadge.js]: Use colors listed here only
https://opensource.box.com/box-ui-elements/#/Colors

File: src/icons/badges/MetadataDefaultBadge.js (hunk 2 above)
Comment (priyajeet) [reply src/icons/badges/MetadataDefaultBadge.js]: See https://opensource.box.com/box-ui-elements/#/Colors for available colors

Comment (ChenCodes) [reply src/icons/badges/MetadataDefaultBadge.js]: I am not seeing that variable name in BUI. Where could I find this variable?

File: src/icons/badges/MetadataActiveBadge.js (hunk 1 above)
Comment (ChenCodes) [reply src/icons/badges/MetadataActiveBadge.js]: I believe this should be $bdl-box-blue instead of #4D91E2, will make the change for that

Comment (booc0mtaco) [reply src/icons/badges/MetadataActiveBadge.js]: Hm, this color doesn't appear to be one of our main hues. can you attach a screenshot of the affected icons?

Comment (booc0mtaco) [reply src/icons/badges/MetadataActiveBadge.js]: Ah, from the icon this should be box blue. good catch

**Pull request merged**